contrib/znc/files/znc.user

```diff
@@ -0,0 +1,4 @@
+type = process
```

Comment:
what is the package for documentation on tmpfiles? it says to look at tmpfiles.d(5) but man 5 tmpfiles.d didnt work

Reply:
no package holds that because @q66 forgot to add it to sd-utils manpages prolly, but merely googling it gives you the fdo html page for it

its not very useful in this case, since all you need is already shown in the tmpfiles/sysusers files in cports, and most of the features are not used so it would just be confusing